File: Liguagem C#/1015.cs
using System; 

class URI {

    static void Main(string[] args) { 
    string[] input;
    
    input = Console.ReadLine().Split(' ');
    
    double X1 = double.Parse(input[0]);
    double Y1 = double.Parse(input[1]);
    
    input = Console.ReadLine().Split(' ');
    
    double X2 = double.Parse(input[0]);
    double Y2 = double.Parse(input[1]);
    
    double distancia = ((X2 - X1)*(X2 - X1)) + ((Y2 - Y1)*(Y2 - Y1));
    
    Console.Write("{0:F4}\n",Math.Sqrt(distancia));
    
    }

}

File: Linguagem Dart/1015.dart
import 'dart:io';
import 'dart:math';

void main()
{
    List <String> linha1 = stdin.readLineSync().split(" ");
    
    double X1 = double.parse(linha1[0]);
    double Y1 = double.parse(linha1[1]);
    
    List <String> linha2 = stdin.readLineSync().split(" ");
    
    double X2 = double.parse(linha2[0]);
    double Y2 = double.parse(linha2[1]);
    
    double distancia = sqrt(pow((X2 - X1), 2.0) + pow((Y2 - Y1), 2.0));
    
    print(distancia.toStringAsFixed(4));
}
